c/tests/integration/common.c

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright (C) 2022 Vaticle
+ *
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+#include <stdio.h>
+
+#include "c/typedb_driver.h"
+#include "common.h"
+
+const char* TYPEDB_CORE_ADDRESS = "127.0.0.1:1729";
+
+bool print_error(const char* filename, int lineno) {
```

Comment:
This name implies to me that you've already called `check_error()`. Which, reading ahead, you usually do.
Suggestion: call this function `failed()` and structure the tests like so:
```c
Type t = do_a_thing();
if (failed(__FILE__, __LINE__)) goto cleanup;
```
This way you'll also get more useful file:line output rather than it always pointing you to the cleanup block.